tools/mksurfdata_esmf/src/mksoiltexMod.F90

```diff
@@ -165,6 +168,8 @@ subroutine mksoiltex(file_mesh_i, file_mapunit_i, file_lookup_i, mesh_o, pioid_o
     do ni = 1,ns_i
        if (mapunit_i(ni) == 0.) then
           mask_i(ni) = 0
+       else
+          mask_i(ni) = 1
```

Comment:
This turned out not to fix the problem, and may not have any impact at all, but it still fixes something that's an issue in theory.

Reply:
Since mask_i isn't set to anything specific outside of this, this is just good practice of making sure it's set to something everywhere. I think the only problem would be if the compiler set it to 0 rather than a random garbage value. But, yes best practice to make sure it's set to something.